salt/proxy/init.sls

```diff
@@ -1,17 +1,18 @@
-{% if salt.caasp_pillar.get('proxy:systemwide') %}
-  {% set proxy_http  = salt.caasp_pillar.get('proxy:http') %}
-  {% set proxy_https = salt.caasp_pillar.get('proxy:https') %}
-
-  {% set no_proxy = [salt.caasp_pillar.get('dashboard'), '.infra.caasp.local', '.cluster.local'] %}
-  {% set extra_no_proxy = salt.caasp_pillar.get('proxy:no_proxy') %}
-  {% if extra_no_proxy %}
-    {% do no_proxy.append(extra_no_proxy) %}
-  {% endif %}
+{%- if salt.caasp_pillar.get('proxy:systemwide') %}
```

Comment:
why the `-` added in this context

Reply:
For removing the superfluous whitespaces in the rendered file: it makes logs really ugly. I know, it is a bit paranoic...

Reply:
aha ok good :small_airplane: